src/components/Shared/Filters/FilterListItemMultiselect.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Checkbox,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+} from '@material-ui/core';
+import { isArray } from 'lodash';
+import React from 'react';
+import { Filter } from './Filter';
+
+interface Props {
+  filter: Filter;
+  selected;
+  onUpdate: (value) => void;
+}
+
+export const FilterListItemMultiselect: React.FC<Props> = ({
+  filter,
+  selected,
+  onUpdate,
+}: Props) => {
+  const isChecked = (value: string) =>
+    isArray(selected)
+      ? selected.findIndex((it) => it == value) != -1
```

Comment:
Doesn't appear that you care about the index so https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some would be better.
```suggestion
      ? selected.some((it) => it === value)
```